ipc-client/main.c

```diff
@@ -38,6 +38,9 @@ static void quit_herbstclient(int signal) {
 }
 
 void init_hook_regex(int argc, char* argv[]) {
+    if (argc == 0) {
```

Comment:
Or should I be sure and make it <= 0? It is a signed int after all

Reply:
Yes, I think `<= 0` is better.